src/show/pages/private/Admin/index.jsx

```diff
@@ -50,6 +51,10 @@ const Admin = () => {
     country: '',
     date: ''
   });
+  const dispatch = useDispatch();
+
+  const events = useSelector(state => state.event.events);
```

Comment:
```suggestion
  const events = useSelector(state => state.events.events);
```

Your reducer is now at plural (you decided it, but not finished modifying everything)

Reply:
done